my-react-app/src/App.js

```diff
@@ -0,0 +1,15 @@
+import "./App.css";
+import MainContent from "./MainContent";
+
+function App() {
+  return (
+    <div className="App">
+      <div>
+        <h1>Hello, React!</h1>
+        {MainContent()}
```

Comment:
We need to avoid using react component as a function, we need to use it as jsx mark-up like ```<MainContent />```